src/math/camera.rs

```diff
@@ -0,0 +1,49 @@
+use glam::{Mat4, Vec2, Vec3};
+
+pub struct Camera {
+    pub position: Vec2,
+    pub rotation: f32,
+    pub scale: Vec2,
+}
+
+impl Default for Camera {
+    fn default() -> Self {
+        Self {
+            position: Vec2{ x: 0.0, y: 0.0 },
+            rotation: 0.0,
+            scale: Vec2::ONE
+        }
+    }
+}
+
+impl Camera {
+
+    /// Gets the real size of the viewport
+    pub fn get_real_size(&mut self, viewport: Vec2) -> Vec2 {
+        Vec2 {
+            x: viewport.x/self.scale.x, 
+            y: viewport.y/self.scale.y,
```

Comment:
I recommend you format the code with `cargo fmt`, or CTRL+SHIFT+I since you use VSCode.
```suggestion
            x: viewport.x / self.scale.x, 
            y: viewport.y / self.scale.y,
```